AWS/Renew_SSL_Certificate.ipynb

```diff
@@ -1,556 +1,682 @@
 {
- "cells": [
-  {
-   "cell_type": "markdown",
-   "id": "b18495bb-19ba-4b43-9824-8739dd304b90",
-   "metadata": {
-    "jupyter": {
-     "source_hidden": false
-    },
-    "name": "Check SSL Certificate Expiry",
-    "orderProperties": [],
-    "tags": [],
-    "title": "Check SSL Certificate Expiry"
-   },
-   "source": [
-    "<img src=\"https://unskript.com/assets/favicon.png\" alt=\"unSkript.com\" width=\"100\" height=\"100\"/> \n",
-    "<h1> unSkript Runbooks </h1>\n",
-    "<div class=\"alert alert-block alert-success\">\n",
-    "    <b> This runbook demonstrates how to check SSL Certificate expiration date and send a reminder message to the Slack channel using unSkript legos.</b>\n",
-    "</div>\n",
-    "\n",
-    "<br>\n",
-    "\n",
-    "<center><h2>Renew SSL Certificate</h2></center>\n",
-    "\n",
-    "# Steps Overview\n",
-    "    1) Fetch SSL Certificate expiration date from the AWS ACM (Amazon Certificate Manager).\n",
-    "    2) Send a message to Slack ONLY if the number of days remaining are less than 30 days."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "c52f7b11-cca5-4bde-8641-995f5c9e2f43",
-   "metadata": {
-    "local_metadata": {
-     "jupyter": {
-      "source_hidden": false
-     },
-     "name": "Check SSL Certificate Expiry Date",
-     "orderProperties": [],
-     "tags": [],
-     "title": "Check SSL Certificate Expiry Date"
-    },
-    "remote_metadata": {
-     "name": "Check SSL Certificate Expiry Date",
-     "orderProperties": [],
-     "tags": [],
-     "title": "Check SSL Certificate Expiry Date"
-    }
-   },
-   "source": [
-    "In this lego we will check the expiration date of the SSL Certificate and return the number of days remaining before renewal"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "ff009de1-fce8-42b0-823b-1e04c6ac32f8",
-   "metadata": {
-    "local_metadata": {
-     "accessType": "ACCESS_TYPE_UNSPECIFIED",
-     "actionBashCommand": false,
-     "actionNeedsCredential": true,
-     "actionRequiredLinesInCode": [],
-     "actionSupportsIteration": true,
-     "actionSupportsPoll": true,
-     "action_uuid": "afacde59-a401-4a8b-901d-46c4b3970b78",
-     "continueOnError": false,
-     "createTime": "2022-07-27T16:51:48Z",
-     "credentialsJson": {},
-     "currentVersion": "v0.0.0",
-     "description": "Test",
-     "execution_data": {
-      "last_date_success_run_cell": "2022-09-17T14:00:45.439Z"
-     },
-     "id": 100001,
-     "index": 100001,
-     "inputData": [
-     ],
-     "inputschema": [
-      {
-       "properties": {
-        "aws_certificate_arn": {
-         "default": "",
-         "description": "Certificate ARN",
-         "title": "aws_certificate_arn",
-         "type": "string"
+    "cells": [
```

Comment:
I have given comments in the ReviewNB. Could you please take care of the comments

Reply:
Resolved